### DISCUSSION TITLE
Keyboard Layout

### DISCUSSION THREAD

**Discussion by floatn** [tamlok/vnote#1753] [Q&A] [answered]
Hi!
I am using IBus to manage my keyboard layout, and I can't change my layout while VNote window is active.
Please, help.

**Comment by tamlok** [comment, marked as answer]
Does other Qt5 apps work well? Please make sure IBus for Qt is configured correctly: https://wiki.archlinux.org/index.php/IBus

**Reply by floatn** [reply]
That was IBus configuration problem. The link helped. Thank you.
